benchmarks/benchmark_runner.py

```diff
@@ -116,11 +123,8 @@ async def compare_diffs(actual, generated):
     return await grade(prompt, comparison_prompt)
 
 
-async def grade_and_clean_diff(repo, response, result, comparison_diff=None):
+async def grade_and_clean_diff(diff, response, result, comparison_diff=None):
```

Comment:
This doesn't clean anymore. I think we should rename to `grade_diff`. I like the change to having setup clean instead of teardown. It's good to have the benchmark repo easily inspectable after a benchmark.